lmdeploy/archs.py

```diff
@@ -137,6 +133,9 @@ def check_vl_llm(config: dict) -> bool:
 
 def get_task(model_path: str):
     """get pipeline type and pipeline class from model config."""
+    from lmdeploy.serve.async_engine import AsyncEngine
```

Comment:
When I use `python3 -m lmdeploy serve api_server workspace`, `workspace` is obtained through LLM convert and will still depend on it. You can try uninstalling torchvision locally to verify.

Reply:
resolved